newdle/client/src/reducers/creation.js

```diff
@@ -84,7 +84,12 @@ export default combineReducers({
       case ADD_PARTICIPANTS:
         return [...state, ...action.participants];
       case REMOVE_PARTICIPANT:
-        return state.filter(p => p.id !== action.participant.id);
+        if (action.participant.id) {
+          return state.filter(p => !p.id || p.id !== action.participant.id);
```

Comment:
Why not just?
```suggestion
          return state.filter(p => p.id !== action.participant.id);
```

Reply:
newly added participants from user search don't have one

Reply:
But wouldn't in that case `p.id` still be different from `action.participant.id`? As we never have a `undefined` `action.participant.id`.

Reply:
It would be, and thus remove all those participants

Reply:
ok nevermind, i was somehow thinking the other way round. you were right that the first half of that check isn't needed.